daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -95,6 +95,48 @@ For a new Dapr deployment, HA mode can be set with both:
 
 For an existing Dapr deployment, [you can enable HA mode in a few extra steps]({{< ref "#enabling-high-availability-in-an-existing-dapr-deployment" >}}).
 
+## Cluster Critical Priority Class Name
+
+In some scenarios, nodes may have memory and/or cpu pressure and the Dapr control plane pods might get selected
+for eviction. To prevent this, you can set a critical priority class name for the Dapr control plane pods. This ensures that
+the Dapr control plane pods are not evicted unless all other pods with lower priority are evicted.
+
+Learn more about [Protecting Mission-Critical Pods](https://kubernetes.io/blog/2023/01/12/protect-mission-critical-pods-priorityclass/).
+
+There are two built-in critical priority classes in Kubernetes:
+- `system-cluster-critical`
+- `system-node-critical` (highest priority)
+
+It's recommended to set the `priorityClassName` to `system-cluster-critical` for the Dapr control plane pods.  
+
+For a new Dapr deployment, `system-cluster-critical` priority class mode can be set via the helm value `global.priorityClassName`.
+
+For a new Dapr deployment, this priority class can be set with both the Dapr CLI and Helm charts, 
+using the helm `--set global.priorityClassName=system-cluster-critical` argument.
+
+#### Dapr version < 1.14
```

Comment:
Question. Why version 1.14 and below? What is different from 1.14 onwards, is this set for you by default? It does not say.

Reply:
the 1.14 chart adds this [automatically](https://github.com/dapr/dapr/commit/199bcf4e198b247bf0f259562e02c97064cb378e) for the user